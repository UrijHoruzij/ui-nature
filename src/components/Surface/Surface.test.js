import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import { Surface } from "../";
import ThemeProvider from "../../ThemeProvide";

describe("Surface component", () => {
  test("Render surface", () => {
    const component = (
      <ThemeProvider>
        <Surface></Surface>
      </ThemeProvider>
    );
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
