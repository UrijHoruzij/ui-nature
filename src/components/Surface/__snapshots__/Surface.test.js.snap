// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Surface component Render surface 1`] = `
.c0 {
  border-radius: 4px;
  background-color: #ffffff;
  padding: 8px;
  -webkit-filter: drop-shadow(0px 10px 25px rgba(0,0,0,0.1));
  filter: drop-shadow(0px 10px 25px rgba(0,0,0,0.1));
  min-width: 80px;
  min-height: 80px;
  width: 80px;
  height: 80px;
}

<div
  className="c0"
/>
`;
